google/cloud/spanner/samples/samples.cc

```diff
@@ -139,30 +139,74 @@ void DropDatabase(std::vector<std::string> const& argv) {
   (argv[0], argv[1], argv[2]);
 }
 
+void InsertData(std::vector<std::string> const& argv) {
+  if (argv.size() != 3) {
+    throw std::runtime_error(
+        "insert-data <project-id> <instance-id> <database-id>");
+  }
+
+  // TODO(#377) - cache the client across sample functions.
+  namespace spanner = google::cloud::spanner;
+  spanner::Client client(spanner::MakeConnection(
+      spanner::MakeDatabaseName(argv[0], argv[1], argv[2])));
+
+  //! [START spanner_insert_data]
+  namespace spanner = google::cloud::spanner;
+  [](spanner::Client client) {
+    auto insert_singers = spanner::InsertMutationBuilder(
+                              "Singers", {"SingerId", "FirstName", "LastName"})
+                              .EmplaceRow(1, "Marc", "Richards")
+                              .EmplaceRow(2, "Catalina", "Smith")
+                              .EmplaceRow(3, "Alice", "Trentor")
+                              .EmplaceRow(4, "Lea", "Martin")
+                              .EmplaceRow(5, "David", "Lomond")
+                              .Build();
+
+    auto insert_albums = spanner::InsertMutationBuilder(
+                             "Albums", {"SingerId", "AlbumId", "AlbumTitle"})
+                             .EmplaceRow(1, 1, "Total Junk")
+                             .EmplaceRow(1, 2, "Go, Go, Go")
+                             .EmplaceRow(2, 1, "Green")
+                             .EmplaceRow(2, 2, "Forever Hold Your Peace")
+                             .EmplaceRow(2, 3, "Terrified")
+                             .Build();
+
+    auto commit_result = client.Commit(spanner::MakeReadWriteTransaction(),
+                                       {insert_singers, insert_albums});
```

Comment:
Consider moving the `insert_singers` and `insert_albums` since it looks like we're done w/ them.